draft-demarco-oauth-status-assertions.md

```diff
@@ -235,6 +235,9 @@ such continuous surveillance.
 The general requirements for the implementation of Status Assertion are
 listed in this section. The Status Assertion:
 
+- notifies the Holder about the status of their Digital Credential,
+because the Holder MUST be informed by the Issuers of any changes
```

Comment:
this BCP14 keyword would probably be better in a section describing how the issuer conforms to the spec, and not in general requirements, but its ok here.

Reply:
something like: X MUST Y because blah blah

ok, I can do this.

Reply:
done